Documentation/spelling_wordlist.txt

```diff
@@ -370,6 +370,7 @@ extendable
 extensibility
 externalIPs
 fallback
+favour
```

Comment:
Nit: did you try `favor` instead (US English), which would maybe spare us the addition of the word to this list?